docs/user/downloadable-documentation.rst

```diff
@@ -1,50 +1,66 @@
+Understanding Offline Formats
+=============================
 
-Read the Docs supports building multiple formats for Sphinx-based projects:
+This page will provide an overview of a core Read the Docs feature: building docs in multiple formats.
+
+Read the Docs supports the following formats by default:
 
 * PDF
 * ePub
 * Zipped HTML
 
-This means that every commit that you push will automatically update your PDFs as well as your HTML.
+This means that every commit that you push will automatically update your offline formats as well as your documentation website.
+
+Use cases
+---------
+
+This functionality is great for anyone who needs documentation when they aren't connected to the internet.
+Users who are about to get on a plane can grab a single file and have the entire documentation during their trip.
+Many academic and scientific projects benefit from these additional formats.
+
+PDF versions are also helpful to automatically **create printable versions of your documentation**.
+The source of your documentation will be structured to support both online and offline formats.
+This means that a documentation project displayed as a website can be downloaded as a PDF,
+ready to be printed as a report or a book.
+
+Offline formats also support having the entire documentation in a single file.
+Your entire documentation can now be delivered as an email attachment,
+uploaded to an eReader,
+or accessed and searched locally without online latency.
+This makes your documentation project **easy to redistribute or archive**.
 
-This is enabled by the :ref:`config-file/v2:formats` key in our config file.
-A simple example is here:
+Accessing offline formats
+-------------------------
 
-.. code-block:: yaml
+You can download offline formats in the :guilabel:`Project dashboard` > :guilabel:`Downloads`:
 
-   # Build PDF & ePub
-   formats:
-     - epub
-     - pdf
+.. image::  /_static/images/guides/offline-formats.jpg
+    :width: 75%
+
+When you are browsing a documentation project,
+they can also be accessed directly from the :doc:`/flyout-menu`.
+
+Examples
+--------
 
 If you want to see an example,
 you can download the Read the Docs documentation in the following formats:
 
-    * `PDF`_ 
-    * `ePub`_ 
-    * `Zipped HTML`_ 
-    
+    * `PDF`_
+    * `ePub`_
+    * `Zipped HTML`_
+
 .. _PDF: https://docs.readthedocs.io/_/downloads/en/latest/pdf/
 .. _ePub: https://docs.readthedocs.io/_/downloads/en/latest/epub/
 .. _Zipped HTML: https://docs.readthedocs.io/_/downloads/en/latest/htmlzip/
 
-Use cases
----------
-
-This functionality is great for anyone who needs documentation when they aren't connected to the internet.
-Users who are about to get on a plane can grab a PDF and have the entire doc set ready for their trip.
-
-The other value is having the entire docset in a single file.
-You can send a user an email with a single PDF or ePub and they'll have all the docs in one place.
+Continue learning
+-----------------
```

Comment:
```suggestion
Continue learning
-----------------

Downloadable documentation formats are built by your documentation framework. Therefore, it's also your framework that decides exactly how each output is built and which formats are supported:

Sphinx
   All output formats are built mostly lossless from the documentation source, meaning that your documentation source (reStructuredText or Markdown/MyST) is built from scratch for each output format.

MkDocs and Docsify + more
   The common case for most documentation frameworks is that several alternative extensions exist supporting various output formats. Most of the extensions export the HTML outputs as another format (for instance PDF) through a conversion process.

Because Sphinx supports the generation of Downloadable formats through an official process, we are also able to support it officially. Other alternatives can also work, provided that you identify which extension you want to use and configure the environment for it to run.
```

Elaborating this comment:

* As mentioned earlier, it seemed misleading to _require_ Sphinx in our How-To since there are other ways to generate downloadable formats. So we removed that part.
* We are awaiting official support from an Environment variable to dictate the expected location of supported output formats. So our efforts here could "make space" for this.
* Ultimately, I thought adding something in the explanation here is appropriate? I thought that it would be great to put under "Continue learning" since learning more probably relies on the documentation framework.

I don't know if what I wrote gives away a slight bias towards Sphinx here, but I think that it really explains why it is that these formats are supported and the others are unspecified.

Consider removing the last paragraph :+1:

Reply:
Edited to add a small caveat, and shipped 👍